core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairScheduler.java

```diff
@@ -37,4 +39,9 @@
      * @param tableReference The table to remove configuration for.
      */
     void removeConfiguration(TableReference tableReference);
+
+    /**
+     * Get a list of the currently scheduled (read-only) repair jobs.
```

Comment:
@return ?

also - what does "(read-only)" mean here? Is it needed?

Reply:
Removing read-only, this returns RepairJobView(immutable view) rather than the actual TableRepairJobs(can perform actions) which was the "read-only" part mentioned.